ProcessLib/ProcessVariable.cpp

```diff
@@ -116,7 +116,7 @@ ProcessVariable::ProcessVariable(BaseLib::ConfigTree const& config,
 }
 
 ProcessVariable::ProcessVariable(ProcessVariable&& other)
-    : _name(std::move(other._name)),
+    : _name(other._name),
```

Comment:
Removing std::move is wrong here. I have no explanation why clang-tidy suggested this fix.
For LineSegment changes it is OK, because there are only POD's moved, but here it is a std::string, which is expensive to copy...
Reverting this change.